Profiles/__init__.py

```diff
@@ -35,14 +35,84 @@
 # Disable SSL certificate verification warning
 requests.packages.urllib3.disable_warnings()
 
+# LDAP
+ldap = csh_ldap.CSHLDAP(app.config['LDAP_BIND_DN'], app.config['LDAP_BIND_PASS'])
+
+from Profiles.utils import before_request
+from Profiles.utils import ldap_get_active_members
+from Profiles.utils import ldap_get_all_members
+from Profiles.utils import ldap_get_member
+from Profiles.utils import ldap_search_members
+from Profiles.utils import ldap_is_active
+from Profiles.utils import ldap_get_eboard
+from Profiles.utils import get_member_info
+from Profiles.utils import _ldap_get_group_members
```

Comment:
Move this to a single line import `from .utils import before_request, ldap_get_active_members, ...`

Also might consider breaking out `utils.py` into `utils.py` and `ldap.py`

Reply:
done, broke ldap functions into ldap.py